AppendBlobReader/src/appendblobreader.json

```diff
@@ -120,7 +131,13 @@
         }
     },
     "variables": {
-        "consumer_roleGuid": "[guid(parameters('sites_blobreaderconsumer_name'), uniqueString(deployment().name, resourceGroup().id))]",
+        "consumer_roleGuid": "[guid(parameters('sites_blobReaderConsumer_name'), uniqueString(resourceGroup().id))]",
+        "blobReaderConsumer_respurceId": "[resourceId(variables('resourceGroupName'),'Microsoft.Web/sites/', parameters('sites_blobReaderConsumer_name'))]",
+        "appendBlobProcessor_roleGuid": "[guid(parameters('sites_appendBlobProcessor_name'), uniqueString(resourceGroup().id))]",
+        "appendBlobProcessor_resourceId": "[resourceId('Microsoft.Web/sites/', parameters('sites_appendBlobProcessor_name'))]",
+        "BlobReader_resourceId": "[resourceId(variables('resourceGroupName'),'Microsoft.Web/sites/', parameters('sites_blobReader_name'))]",
+        "TABLE_NAME": "FileOffsetMap",
```

Comment:
did we also move it to variables in block blob reader ?

Reply:
ref. taken from block blob reader